README.md

```diff
@@ -193,6 +181,21 @@ In order to run it:
     make runserver
     ```
 
+#### Environment Configuration
+
+By default, `make` targets will use the `cms.settings.dev` settings unless their commands explicitly use a different setting (via the `--settings` parameter or `DJANGO_SETTINGS_MODULE` environment variable).
+is set in the environment. This default should work out of the box for local development.
+
+To override settings in the environment, you can use a `.env` file. Note, however, that settings this file may also be picked up in the docker container, so
+you may need to remove or rename the file, or comment out specific variables if you switch to running the app in the container.
+
+> [!NOTE]
+> When running the application locally in a virtual environment via Poetry the `.env` file will not be picked up automatically.
+> For this to work you'll need to install the [poetry-plugin-dotenv](https://github.com/pivoshenko/poetry-plugin-dotenv).
+> However, if you installed Poetry with `brew` rather than `pip` that currently isn't going to
+> work (see the [issue](https://github.com/pivoshenko/poetry-plugin-dotenv/issues/327)) and you'll need to install an older and
+> seemingly no longer maintained [poetry-dotenv-plugin](https://github.com/mpeteuil/poetry-dotenv-plugin).
+
```

Comment:
It should have been done this way from the start 😄 
I remember I started with this approach, but for a reason I don't recall now I shifted to doing it via the `.env` file.

Everything looks good, but we might want to get rid of the bit I've highlighted since we're no longer using the `.env` file and it won't be relevant for most people unless someone decides to use their own `.env` (but again it'd now be easier to modify the settings files..)
Not a huge issue tho as it looks that others are fine with it!

Reply:
I think the note is useful. Let's leave it in and revisit should someone new try this and stumble.